js/main.js

```diff
@@ -264,14 +264,18 @@ function initMarker(feature) {
     } else {
         where = '';
     }
+
+    var additionalInformationLink = properties.details_URL ?
+        '<span><a href='+properties.details_URL+'> weitere Informationen </a></span>' : '';
+
```

Comment:
:+1: I like that no HTML element is added if the attribute is missing. Does it also account for an empty string?

Reply:
Yes

```Boolean('')```
> false